arex-storage-web-api-service/src/main/java/com/arextest/storage/web/api/service/bean/StorageCacheConfiguration.java

```diff
@@ -34,6 +39,20 @@ CacheProvider cacheProvider() {
         return provider;
     }
 
+    @Bean(name = "operationCache")
+    public Cache<String, String> operationCache() {
+        if (maximumWeight == 0) {
+            maximumWeight = 2 * 1024 * 1024 * 1024;
+        }
+        Cache<String, String> cache = Caffeine
+                .newBuilder()
+                .initialCapacity(1000)
+                .maximumWeight(maximumWeight)
+                .weigher((String key, String value) -> key.length() + value.length())
+                .build();
+        return cache;
+    }
+
```

Comment:
There is an instance of CacheProvider used by storage, Any others is unnecessary and redundant.

Reply:
done.
Replacing local cache with Redis